src/einspline/tests/test_3d.cpp

```diff
@@ -86,44 +86,48 @@ TEST_CASE("double_3d_periodic","[einspline]")
   bc.lCode = PERIODIC;
   bc.rCode = PERIODIC;
 
-  UBspline_3d_d* s = create_UBspline_3d_d(grid, grid, grid, bc, bc, bc, data);
+  auto s = std::unique_ptr<UBspline_3d_d, void (*)(void*)>{create_UBspline_3d_d(grid, grid, grid, bc, bc, bc, data),
```

Comment:
It seems working very well. Could you update all the pointers in test_3d.cpp and test_one.cpp?

Reply:
done